lib/puppet/network/auth_config_parser.rb

```diff
@@ -28,11 +28,11 @@ def parse_rights
         right = rights.newright(name, count, @file)
       when /^\s*(allow(?:_ip)?|deny(?:_ip)?|method|environment|auth(?:enticated)?)\s+(.+?)(\s*#.*)?$/
         if right.nil?
-          raise Puppet::ConfigurationError, "Missing or invalid 'path' before right directive at line #{count} of #{@file}"
+          raise Puppet::ConfigurationError, _("Missing or invalid 'path' before right directive at line %{count} of %{file}") % { count: count, file: @file }
```

Comment:
Can you add a note that `path` is a reserved keyword here (referring to an entry in the authorization config file) and should not be translated?